.github/workflows/services-json.yml

```diff
@@ -0,0 +1,36 @@
+name: Services Validator
+
+on:
+  push:
+    paths-ignore:
+      - "cmake/**"
+    branches-ignore: [master]
```

Comment:
Ignoring master branch is probably not desired. Since it's the only one we really use for active development.